.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,14 +1,39 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a report with information to help us assist you
 title: ''
-labels: ''
+labels: 'Bug Report'
```

Comment:
this label does not currently exist. did you mean to use the existing `bug` label, or create a new one?